maskit/examples/load_data.py

```diff
@@ -0,0 +1,14 @@
+from maskit.examples.load_circles import load_circles
+from maskit.examples.load_iris import load_iris
+from maskit.examples.load_mnist import load_mnist
+
+
+def load_data(dataset, wires=None, embedding=None, params=None):
+    if dataset == "iris":
+        return load_iris(wires, params)
+    elif dataset == "mnist":
+        return load_mnist(wires, params)
+    elif dataset == "circles":
+        return load_circles(wires, params)
+    else:
+        return [None, None, None, None]
```

Comment:
I would raise an error if the dataset is "unknown". Usually, that indicates a user error which should not pass silently.
```suggestion
        raise ValueError(f"unknown dataset {dataset}")
```
Is ``[None, None, None, None]`` actually a sensible value for the client code?

Reply:
In the old main there is a dataset called "simple" that somehow works with `[None, None, None, None]`. But I agree that an error is probably the better way for completely unknown datasets.

Reply:
The "simple" dataset just learns to go back to the |0> state from a fixed state. Therefore, we don't have any training or test samples.